includes/blocks/podcast/markup.php

```diff
@@ -40,7 +40,11 @@
 $episode_type   = get_post_meta( $post_id, 'podcast_episode_type', true );
 $episode_number = get_post_meta( $post_id, 'podcast_episode_number', true );
 $season_number  = get_post_meta( $post_id, 'podcast_season_number', true );
-$term_image_id  = get_term_meta( $podcast_show->term_id, 'podcasting_image', true );
+if ( is_a( $podcast_show, 'WP_Term' ) ) {
```

Comment:
Note that this solves the undefined issue but doesn't actually solve the root problem, where the proper data isn't actually set on any existing blocks, even if the data was there before. I'm fine with adding this defensive check but just want to ensure we don't lose the thread on the larger issue